pkgs/by-name/ro/root/package.nix

```diff
@@ -165,18 +165,15 @@ stdenv.mkDerivation rec {
   cmakeFlags =
     [
       "-DCLAD_SOURCE_DIR=${clad_src}"
-      "-DCMAKE_INSTALL_BINDIR=bin"
-      "-DCMAKE_INSTALL_INCLUDEDIR=include"
-      "-DCMAKE_INSTALL_LIBDIR=lib"
       "-DClang_DIR=${clang}/lib/cmake/clang"
       "-Dbuiltin_clang=OFF"
       "-Dbuiltin_llvm=OFF"
       "-Dfail-on-missing=ON"
       "-Dfftw3=ON"
       "-Dfitsio=OFF"
-      "-Dgnuinstall=ON"
```

Comment:
If we are already setting the directories, why is it bad to still pass gnuinstall? Do you have a comparison of the directory structures before and after this change?